src/client/containers/ProductPage/ProductPage.Container.js

```diff
@@ -16,14 +16,22 @@ const ProductPageContainer = ({ isAuthenticated }) => {
   const [product, setProduct] = React.useState({});
   const [category, setCategory] = React.useState('');
   const [similarProduct, setSimilarProduct] = React.useState([]);
+  const [fav, setFav] = React.useState(false);
 
   const productData = useFetchApi(`products/${id}`);
+  const favsData = useFetchApi(`users/${auth.currentUser.uid}/${id}/favorites`);
 
   React.useEffect(() => {
-    if (!productData.isLoading) {
+    if (!productData.isLoading && !favsData.isLoading) {
       setProduct(productData.data[0]);
+      if (!favsData.data.error) {
+        const result = favsData.data.some((item) => {
+          return item.id == id;
```

Comment:
Change line 29 to : `return item.id === id;` to remove eslint error

Reply:
used '==', since the type is not same. Now changed code to item.id === parseInt(id,10);